service/resource/deployment/deployment.go

```diff
@@ -0,0 +1,11 @@
+package deployment
+
+const (
+	clusterSetupDeploymentName = "cluster-setup"
+)
+
```

Comment:
Here we return the list of deployments created by the operator. For now only cluster setup is added.